sql/core/src/test/resources/sql-tests/results/group-by.sql.out

```diff
@@ -1071,6 +1071,39 @@ struct<k:int,regr_syy(y, x):double>
 2	200.0
 
 
+-- !query
+SELECT regr_slope(y, x) FROM testRegression
```

Comment:
nit: can we have a new golden file for these linear regression functions?

Reply:
OK